subnet/subnet_test.go

```diff
@@ -39,6 +39,7 @@ func newDummyRegistry() *MockSubnetRegistry {
 		{ip.IP4Net{ip.MustParseIP4("10.3.2.0"), 24}, attrs, exp, 11},
 		{ip.IP4Net{ip.MustParseIP4("10.3.4.0"), 24}, attrs, exp, 12},
 		{ip.IP4Net{ip.MustParseIP4("10.3.5.0"), 24}, attrs, exp, 13},
+		{ip.IP4Net{ip.MustParseIP4("10.3.31.0"), 24}, attrs, exp, 13},
```

Comment:
can we add a comment for why 10.3.31.0 is here?

For example: a hand created lease outside the range of subnetMin-SubnetMax for testing removal.